### DISCUSSION TITLE
Requirements for GA

### DISCUSSION THREAD

**Discussion by cyrille-leclerc** [jenkinsci/opentelemetry-plugin#50] [General]
What are our requirements for GA?

I propose:

Requirements for GA:
* Reliability: don't break Jenkins, don't break the builds, don't pollute the logs
* No additional feature is required for GA
* Major usability bugs

NOT Requirements for GA
* Scalability: we have not identified scalability risks and the behaviours seems to be ok. We postpone  scalability tests.
* New features.

**Comment by v1v** [comment]
We can discard the cloud provider for the GA, right?

Regarding the requirements for GA:
- Refactor the `README.md` in sections
- Add examples with docker-compose for the existing integrations? That's something I was thinking then people can easily understand how to plug all the bits and pieces together.


Other than that 💯 , I'm really excited about this project!! ❤️

**Reply by cyrille-leclerc** [reply]
Good catch for the `README.md`. 
I would say that the docker compose is a nice to have but not a requirement, I'm not even sure which component I would put in this docker compose :-) .
